app/assets/stylesheets/layout.scss

```diff
@@ -2116,17 +2116,16 @@ table {
     padding: $line-height / 4;
     position: relative;
 
-    .relative,
-    [class^="icon-arrow"] {
-      padding-left: $line-height / 2;
+    .relative {
+      padding-left: rem-calc(18);
     }
 
-    [class^="icon-arrow"] {
-      font-size: $base-font-size;
-      left: -20px;
+    [class^="far"] {
```

Comment:
I guess here we mean `.far` (correct me if I'm wrong). Otherwise we might accidentally target classes starting with `far` (which isn't a common way to start a word, but you never know, particulary with all these modern class names :smile:).

By the way, ideally we wouldn't have to change the view in order to change an icon and changing CSS would be enough, but that's another story :wink:.

Reply:
Good one! 😌